sdk/nodejs/examples/pastebin/src/index.ts

```diff
@@ -0,0 +1,117 @@
+const { Command } = require("commander");
+const zrok = require("zrok")
+const ziti =  require('@openziti/ziti-sdk-nodejs')
+const express = require('express')
+var readlineSync = require('readline-sync');
+import fs from "node:fs"
+
+
+const program = new Command();
+
+program
+  .command('copyto')
+  .version("1.0.0")
+  .description("command to host content to be pastedfrom'd")
+  .action(async () => {
+    var data = readlineSync.question('Input some text... ');
+
+    console.log("data is: ", data)
+    let root = zrok.Load()
+    await zrok.init(root);
+    //await ziti.init( root.env.ZitiIdentity ).catch(( err: Error ) => { console.error(err); return process.exit(1) });
+    ziti.setLogLevel(0)
+    console.log("setting up zrok.CreateShare...")
+    let shr = await zrok.CreateShare(root, new zrok.ShareRequest(zrok.TCP_TUNNEL_BACKEND_MODE, zrok.PUBLIC_SHARE_MODE, "pastebin", ["public"]));
```

Comment:
`tcpTunnel` and `public` is an invalid state. This should be `private`. The controller doesn't prevent it, but it's incorrect.

Reply:
fixed